packages/jsx-email/src/renderer/render.ts

```diff
@@ -75,6 +76,7 @@ const processHtml = async (config: JsxEmailConfig, html: string) => {
     '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">';
   const movePlugin = await getMovePlugin();
   const rawPlugin = await getRawPlugin();
+  const conditionalPlugin = await getConditionalPlugin();
   const settings = { emitParseErrors: true };
   const reJsxTags = new RegExp(`<[/]?(${jsxEmailTags.join('|')})>`, 'g');
```

Comment:
leverage the rawPlugin and conditionalPlugin to remove the jsx-email-raw and jsx-email-cond tags, instead of using a regex to remove them.